### PR TITLE
Added scripts for deploy devlab, clean it and run migration

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

File: devlab/utils/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+cur_pwd=`pwd`
```
Comment (roman-verchikov) [inline devlab/utils/deploy.sh]: unused variable

File: devlab/utils/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+source .env/bin/activate
```
Comment (roman-verchikov) [inline devlab/utils/deploy.sh]: This will break for people using venvs in other dirs

File: devlab/utils/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export DST_OS_NEUTRON_ENDPOINT="http://$DST:9696/"
+cd devlab
+
+echo "Destroy the lab"
+vagrant destroy -f grizzly icehouse
+
+echo "Create the lab"
+vagrant up grizzly icehouse
+sleep 30
+ssh root@$SRC "for f in \`grep -l -r \"192.168.1.2\" /etc/\`; do sed -i 's/192.168.1.2/$SRC/g' \$f; done; mysqldump -u root -psecret keystone > ./db.sql; sed -i 's/192.168.1.2/$SRC/g' db.sql; mysql -u root -psecret keystone < ./db.sql"
```
Comment (roman-verchikov) [inline devlab/utils/deploy.sh]: Please make it multiline for better readability

File: devlab/utils/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+echo "Generate load on SRC"
+cd tests
+python generate_load.py
+ssh root@$SRC "export OS_TENANT_NAME=admin;export OS_USERNAME=admin;export OS_PASSWORD=admin;export SERVICE_TOKEN=ADMIN;export OS_AUTH_URL=\"http://$SRC:5000/v2.0/\";export SERVICE_ENDPOINT=\"http://$SRC:35357/v2.0\"; $extra_cmds"
+
+#below is needed for full cleaning but it's not needed for new deploy
+#cd tests && python generate_load.py --clean --env SRC
+#cd tests && python generate_load.py --clean --env DST
+#ssh root@$SRC "mysql -u root -psecret cinder -e \"delete from cinder.volumes;\""
+#ssh root@$DST "mysql -u root -psecret cinder -e \"delete from cinder.volume_admin_metadata; delete from cinder.volumes;\""
```
Comment (roman-verchikov) [inline devlab/utils/deploy.sh]: remove these, if not needed

File: devlab/utils/os.sh [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+declare -x OS_USERNAME="admin";
+EOF`
+DST_SSH="${DST_SSH_CMD} ${DST_ENV}"
+SRC_SSH="${SRC_SSH_CMD} ${SRC_ENV}"
+net_service="neutron"
+
+#define functions
+#arg1 - project name
+#arg2 - project description
+function CREATE_TENANT {
```
Comment (roman-verchikov) [inline devlab/utils/os.sh]: Uppercase is usually used for variables in bash. Functions should use lower case letters with underscores. Please reformat

**Comment by roman-verchikov** [comment]
1. Put meaningful description into commit message and pull request title
2. Add documentation to each script
    ```
    Description
    <...>
    Usage
    <...>
    ```
3. Remove unused commented out code
4. Let's use `$()` instead of `

**Comment by miarmak** [comment]
Agree with Roman about renaming files and adding documentation.

**Comment by roman-verchikov** [comment]
Looks ok

**Pull request merged**